### ISSUE TITLE
Resolve "Image has non-empty accessible name" feedback

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act#465]
### Charu

- [x]  Images with alt="" will pass the rule, they do not get an implicit role of none per chrome a11y tree. It just says accessibility node not exposed. Should that be clarified under Expectation?Also, image with the title and no alt passes the rule. Can that be clarified under Expectations as well?

- [x]  alt="" does not imply role of none per https://www.w3.org/TR/html-aria/#docconformance saya no corresponding role

**Answer**: html-aria is inconsistent with html-aam. html-aam is the spec that defines how this should work, which says images with an empty alt have role=presentation. No changes needed.

### Kathy

- [x] Passed Example 6: This img element has an explicit role of presentation because of the value of the role attribute - shouldn't this be inapplicable since it's not in accessibility tree due to role=presentation? Same for Passed Ex 5 and Passed 7

### Mary Jo

- [x] This is the first time I focused in on this, but I'm finding it odd having the requirements mapping have references that are techniques, not WCAG SC - especially when they are also listed in the Background section. WCAG techniques themselves are not "required". Even Failure techniques are not themselves "requirements" but supporting info regarding the requirement. I notice a lot of the rules are now having techniques listed in the Requirements section, but I'd like to understand why.

**Answer**: The TF discussed this. Using techniques here is correct. No change needed.

- [x] There are some typos:
  - Need a period at the end of the sentence in the description section
  - Accessibility support section, last bullet, second sentence: "does not specifies" should read "does not specify".

- [ ]  Another thing that is becoming increasingly noticeable to me is the size of the glossary with the addition of examples and links and all that other stuff makes the rule very long and the sections beyond it much more tedious to reach. Should we have a glossary page that includes all glossary terms and exhaustive info and then only provide the terms with a basic definition and link to that info inside of the rules? Just a thought because I think the glossary has now overcome the rest of the content in size.

**Comment by WilcoFiers** [comment]
Everything should be resolved. Ready for another review.

**Comment by maryjom** [comment]
See [survey results](https://www.w3.org/2002/09/wbs/93339/ACTImageAccName3/results) due 20 August.

**Comment by maryjom** [comment]
Awaiting pull request to make small change by @WilcoFiers. Then we can put out for CfC.

**Comment by WilcoFiers** [comment]
Updates have been made, this one can go into a CfC

**Comment by WilcoFiers** [comment]
This shouldn't be open anymore. This rule was published in October 2020.
